docs/configuration/http_filters/fault_filter.rst

```diff
@@ -46,6 +46,7 @@ including the router filter.*
           "fixed_delay_percent" : "...",
           "fixed_duration_ms" : "..."
         },
+        "upstream_cluster" : "...",
```

Comment:
So, this is restricting faults to a specific upstream cluster. And not necessarily to a specific route. The ability to restrict to a route (or authority headers), is already there. 

Put another way, the best way to view is in the context of weighted clusters #247 , where there are multiple clusters for a route.